code/game/objects/machinery/cryopod.dm

```diff
@@ -1,3 +1,12 @@
+#define HS_CNSL_GUN 0
```

Comment:
Abbreviations for the sake of abbreviation makes code less readable.
Please just use the full words so it's less insane